app/src/main/kotlin/com/wire/android/util/time/ISOFormatter.kt

```diff
@@ -0,0 +1,21 @@
+package com.wire.android.util.time
+
+import android.content.Context
+import android.text.format.DateFormat
+import dagger.hilt.android.qualifiers.ApplicationContext
+import java.time.Instant
+import java.util.Date
+import javax.inject.Inject
+
+class ISOFormatter @Inject constructor(@ApplicationContext val context: Context) {
+
+    fun fromISO8601ToTimeFormat(utcISO: String): String {
+        val date = Date.from(Instant.parse(utcISO))
+
+        val localDateFormat = DateFormat.getDateFormat(context)
+        val localTimeFormat = DateFormat.getTimeFormat(context)
+
+        return "${localDateFormat.format(date)}, ${localTimeFormat.format(date)}"
```

Comment:
It's not fully locale-dependent.

I think it's better to use `DateTimeFormatter.ofLocalizedDateTime(FormatStyle.MEDIUM)`
and we can just pass the `Instant`:
```
val instant = Instant.parse(utcISO)
val formatter = DateTimeFormatter.ofLocalizedDateTime(FormatStyle.MEDIUM)
return formatter.format(instant)
```

Reply:
true, fixed that